hs_core/tests/test_update_quota_usage.py

```diff
@@ -157,7 +157,7 @@ def test_zipping_files_increase_quota(self):
 
         # Assert that the quota has been updated correctly
         dz = self.convert_gb_to_bytes(user_quota.data_zone_value)
-        size_of_zip = 10240
+        size_of_zip = 145
```

Comment:
@devincowan - the old value doesn't make sense to me, not sure how 3, 27 byte files could be compressed to 10 kb. I pulled this value from the ResourceFile of the zip.

Reply:
I know it's hard to believe but here's a screenshot of the filesizes reported by irods after running the test:
![image](https://github.com/user-attachments/assets/b1f8e662-ae08-495c-badc-872f5c2c6350)

It is ~10kb and if I download the file, filesystem indeed shows 10240 bytes...

Reply:
Maybe the updated zip logic compresses it more efficiently?

Reply:
I guess so, I remember seeing this when I did the quota work and thinking it was probably an irods bug in `ibun`